easybuild/easyconfigs/m/magma/magma-2.5.4-fosscuda-2019b.eb

```
@@ -12,16 +12,21 @@ toolchainopts = {'pic': True, 'openmp': True}
 
 source_urls = ['https://icl.cs.utk.edu/projectsfiles/magma/downloads/']
 sources = [SOURCE_TAR_GZ]
-checksums = ['7734fb417ae0c367b418dea15096aef2e278a423e527c615aab47f0683683b67']
+patches = ['magma-2.5.4_allow-all-sms.patch']
+checksums = [
+    '7734fb417ae0c367b418dea15096aef2e278a423e527c615aab47f0683683b67',  # magma-2.5.4.tar.gz
+    '0de5d7b3de7f53e54aeb309193cce04a6a5b265a494dc5416e18fd7d6734386e',  # magma-2.5.4_allow-all-sms.patch
+]
 
 builddependencies = [
     ('CMake', '3.15.3'),
 ]
 
-local_gpu_target = '-DGPU_TARGET="Kepler Maxwell Pascal Volta"'
+# default CUDA compute capabilities to use (override via --cuda-compute-capabilities)
+cuda_compute_capabilities = ['3.0', '3.5', '5.0', '6.0']  # or 'Kepler Maxwell Pascal Volta'
```

I think it would be best if this list of version numbers matched the string, so, 7.0 for Volta should be included as well. 
Also, while we are at it, could we also add `'Turing'` and `7.5` in the defaults?
```suggestion
cuda_compute_capabilities = ['3.0', '3.5', '5.0', '6.0', '7.0', '7.5']  # or 'Kepler Maxwell Pascal Volta Turing'
```

---

Good catch. The magma default only adds Pascal and that is where I had the default from.
I wouldn't add Turing though as this is an existing EC so I'd keep the changes minimal. But no strong opinion from my side.

---

There is no harm in adding this I think? Noone is required to rebuild, but it's just better default for those who are "late to the party". Besides, it would already change for anyone who uses `--cuda-compute-capabilities`. Since the test-build is quite long, I would rather add it directly in this PR.

Another thought i had was; The "string" version would not be useable here at all right? I can only imagine it totally breaking `cuda_compute_capabilities`. Do we even need it?

---

Done.
And yes that would not be usable. I only left it there as a comment so people know what that is/was. I'd remove the comment in new magma ECs